src/dune/expander.ml

```diff
@@ -29,6 +29,7 @@ type t =
   ; bindings : Pform.Map.t
   ; scope : Scope.t
   ; c_compiler : string
+  ; cflags : string list
```

Comment:
I'd rather just get rid of this field and access it from the  context every time. Same for c_compiler.